desktop_switcher.ahk

```diff
@@ -81,17 +93,19 @@ getSessionId()
 
 _switchDesktopToTarget(targetDesktop)
```

Comment:
At some point we need to come back through and standardize this file. Function naming conventions, organizing related functions near each other, and commenting more thoroughly (to promote contribution). Just a note that we should get to in the future; this PR looks good.

Reply:
We could also make use of `#include functions/file.ahk`, having separate files, not one long one. But either way is fine, really.

Reply:
Yeah, that's fine too (could be nice to hide a few things in another folder). I'm fine either way.

Reply:
And those global functions - they aren't really global
```
mapDekstops()   *sets global vars*
switchToDesktop()  *retrieves global vars*
```
is essentially the same to 
```
context = mapDesktops()
switchToDesktop(context)
```


Also, _it works on my machine_, with Elijas:master being up to date, running the script as Admin

Reply:
If you check it in, and it doesn't work for other people, we can make a bug fix. But since the documentation currently specifies these hotkeys should work, we can probably put this in.

I'm a bit concerned about the hotkeys we're using up though - I'm not sure how much of the ctrl+alt hotkey space will conflict with what we're doing (e.g in onenote, ctrl+alt+2 puts text in 'heading 2' style, and is now overwritten by WDS). We may need to revisit some of this in the near future...

Reply:
Okay, so if I understood correctly, you're giving the green light for a merge ?